pitest/src/test/java/org/pitest/functional/FCollectionTest.java

```diff
@@ -60,9 +60,9 @@ public void shouldApplyForEachToAllItems() {
     final List<Integer> actual = new ArrayList<>();
     final Consumer<Integer> e = a -> actual.add(a);
 
-    FCollection.forEach(this.is, e);
+      this.is.forEach(e);
 
-    assertEquals(this.is, actual);
+      assertEquals(this.is, actual);
```

Comment:
The additional indention is strange ... but doesn't harm anybody ;-) Maybe the whole method is not correctly indented? I didn't check.

Reply:
fixed